CONTRIBUTING.MD

```diff
@@ -0,0 +1,37 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change. 
+
+### <a name="submit-issue"></a> Submitting an Issue
+
+Before you submit an issue, please search the issue tracker, maybe an issue for your problem already exists;
+
+A minimal reproduction allows us to quickly confirm a bug (or point out a coding problem) as well as confirm that we are fixing the right problem.
+
+We will be insisting on a minimal reproduction scenario in order to save maintainers time and ultimately be able to fix more bugs. 
+Unfortunately, we are not able to investigate / fix bugs without a minimal reproduction, so if we don't hear back from you we are going to close an issue that doesn't have enough info to be reproduced.
+
+## Pull Request Process
+
+1. We have a [code review process](), please follow it in all your interactions with the project. //TODO: Add link to code review process
+2. README.md includes all details how to build project, run tests, etc.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
```

Comment:
I'm not sure that we should use SemVer. Let's discuss this on sync up and coding day.